src/UI/Component/Panel/Listing/Appointement.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Panel\Listing;
+
+/**
+ * Interface Appointment
+ * @package ILIAS\UI\Component\Panel\Listing
+ */
+interface Appointment extends \ILIAS\UI\Component\Component {
+	/**
+	 * Gets the title of the appointment listing
+	 *
+	 * @return string
+	 */
+	public function getTitle();
+
+	/**
+	 * Get the link pointing back to target for getting more
+	 * items (e.g. if selected a larger count of days to be displayed)
+	 *
+	 * @return string
+	 */
+	public function getAction();
```

Comment:
@alex40724 : Either that (the renderer is aware, that only a part of the list needs to be rendered OR the js firing the async request picks the correct part of the complete DOM returned. You may look at the current popover proposal. There this is performed this through JS: https://github.com/ILIAS-eLearning/ILIAS/pull/513.